invenio_deposit/config.py

```diff
@@ -33,7 +33,7 @@
 DEPOSIT_SEARCH_API = '/api/deposits'
 """URL of search endpoint for deposits."""
 
-DEPOSIT_RECORDS_API = '/api/deposits/{pid_value}'
```

Comment:
FYI

> `{{ _('Hello %(user)s!')|format(user=user.username) }}`

Should be `{{ _('Hello %(user)s!', user=user.username) }}`

And you can change `config.DEPOSIT_RECORDS_API|format(pid_value=pid.pid_value)` to `config.DEPOSIT_RECORDS_API.format(pid_value=pid.pid_value)`